app/views/shared/check_answers/_partner_details.html.erb

```diff
@@ -0,0 +1,74 @@
+<% read_only = false unless local_assigns.key?(:read_only)
+   heading_class = local_assigns.key?(:heading) ? local_assigns[:heading] : "govuk-heading-m" %>
+<h2 class="<%= heading_class %>"><%= t ".section_heading" %></h2>
+<%= govuk_summary_list(classes: "govuk-!-margin-bottom-9",
+                       html_attributes: { id: "partner-details-questions" },
+                       actions: !read_only) do |summary_list| %>
```

Comment:
> `actions: !read_only`

Uch, why didn't I think of doing this! So much simpler!

Reply:
Sorry! I didn't think to look at how you'd been implementing it - I should probably have matched your pattern to avoid fragmenting the codebase 😬 
Let's have a chat next week :+1: